hack/cape-custom-scripts.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+echo $(pwd)
```

Comment:
prefer to add `set -e` to stop the execution if any of the commands fail.

Reply:
@dinesh  added